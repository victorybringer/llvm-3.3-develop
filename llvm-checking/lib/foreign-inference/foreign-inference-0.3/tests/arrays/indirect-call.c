#include "base.h"

typedef int (*accessor)(int *, int *);

accessor A = oneDimAccess;

int g(int * arr, int * in) {
  return arr[*in + 5];
}

void setup() {
  A = g;
}

int f(int * arr, int * in) {
  return A(arr, in);
}
