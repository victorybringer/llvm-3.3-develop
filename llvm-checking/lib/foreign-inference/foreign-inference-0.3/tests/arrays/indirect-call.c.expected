fromList [ (("@g", "%arr"), 1)
         ]
